### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by BenjaminGarza**

### CODE REVIEW

**Comment by Oghenebrume50** [comment]
Hi @BenjaminGarza  This is great work, I have a few corrections for you to implement though.
 - [x] The NEWSWEEK logo at the top should be centred and there should be a date to the left of it.
- [x] around the 669px width breakpoint the more stories section looks completely off, could you fix that

Great  work so far, make these changes and submit for re-review. BTW I noticed you didn't use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) please check it out but don't bother using it on this project, just keep it in mind for your next HTML projects, that's what you should be using.

**Comment by bolah2009** [comment]
Currently under review

File: Main.css [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+@media only screen and (max-width: 945px){
+    .hide-small{
+        display: none;
+    }
+    .newsweek-logo{
+        font-size: 30px;
+    }
+}
+@media only screen and (min-width: 768px){
+
```
Comment (bolah2009) [inline Main.css]: ### `HINT`

Consider adding a smaller font size for the navigation bar for smaller screens. Also, the footer can use some repositioning for smaller screens. See illustration below:

| Current | Preferred |
| :-----: | :------: |
| bootstrap footer current | bootstrap footer preffered |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,952 @@
...
+                                    globe, as well as one city, which will be chosen as the "smartest city in the
+                                    world."</p>
+
+                            </div>
+                        </article>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col">
+                <h6 class="more-stories-title">MORE STORIES</h6>
```
Comment (bolah2009) [inline index.html:242]: ### `HINT`

Consider styling this and all other occurrences. (For example, giving it font color and weight as in the source page). See below:

| Current | Preferred |
| :-----: | :-----: |
| more stories current | more stories preffered |

#### Do this for all other occurrences.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Almost there  :smile:  

The clone looks nice, however, I would suggest making sure **nothing is messed up** compared to the source page. There are some styling that is not implemented in the clone.
See below (check the full [page](https://www.newsweek.com/) for more section that needs to be worked on:

| Current | Preferred |
| :-----: | :-----: |
| bootstrap full current | bootstrap full preffered |

You can recreate this issue (Image above) by using the console on chrome **(Inspect `Ctrl+Shift+I`)** and the **Toggle device toolbar** `Ctrl+Shift+M`. This way you can simulate how the app will render on small to widescreen devices.


#### Friendly reminder: You should use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in this project. However, changing it now would take too much time. Please make sure you use Github Flow and [Linter](https://github.com/microverseinc/linters-config#linters-config) in the next project. You will find instructions in the Pathwright lesson description.


Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Hi there @BenjaminGarza 
You worked hard making these changes. But I notice that the stickler is not verifying your project. Maybe this is happening because it's not set up properly. Please check the instruction on the pathwrite on how to set up the linter and make it work on this project. You will also need to fix errors(if the linter found any). I know that in the past we didn't pay much attention to it but linting is important to reduce errors and improve the overall quality of your code. Using the linter also gives a professional look in your repository. 

Please kindly make this change and submit for code review again.
Happy coding :smile: 

[FOR FUTURE PROJECT]
 You should use Github/Git Flow in future projects. It's really easy to use it and will make your repository looks more professional. You will find instructions in the Patwright lesson description.

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey @BenjaminGarza @Oghenebrume50  Great! job with the changes requested by @Torres-ssf  and @bolah2009 
image. 

#### There are just a few things to address 
##### The Newsweek logo is too big
fix1

##### The content on the sides of the newsweek logo are to small.
fix2

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — APPROVED** [review]
### Hi there @BenjaminGarza 
I'm giving this project a :+1: 
Go on and merge to master. Good luck with the next one.

Friendly reminder: You should use Github/Git Flow in this project. However, changing it now would take too much time. Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

**Pull request merged**